cosmo_tester/test_suites/bootstrap_based_tests/inplace_upgrade_test.py

```diff
@@ -69,9 +66,10 @@ def test_inplace_upgrade(cfy,
     cfy.snapshots.upload([snapshot_path, '-s', snapshot_name])
     cfy.snapshots.restore([snapshot_name, '--restore-certificates'])
     _wait_for_restore(manager)
-    cfy.agents.install(['-a'])
```

Comment:
This doesn't work with community (because it uses client.tenant.list()), and it doesn't work with premium, because the way the premium test is set up it creates deployments in two tenants, but none in default tenant. Jiras have been created for these issues.

Reply:
Really? I was sure the part where no deps exist on the default tenant was taken care of. But apparently not. Can you link the Jiras here?

Reply:
The no deployments in default (current?) tenant is: https://cloudifysource.atlassian.net/browse/CFY-7214
The one that breaks it for community is addressed by: https://cloudifysource.atlassian.net/browse/CFY-7204